packages/webpack/src/builder.js

```diff
@@ -60,6 +65,18 @@ export default class WebpackBuilder {
       }
     }
 
+    // Check styleResource existence
+    const styleResources = this.context.options.build.styleResources
```

Comment:
@clarkdo As this was a loader, I've just kept in the place as original PR was. What do you think?